tests/pfcwd/conftest.py

```diff
@@ -244,3 +244,31 @@ def pfcwd_pause_service(ptfhost):
         needs_resume["garp_service"] = False
 
     logger.debug("pause_service needs_resume {}".format(needs_resume))
+
+
+@pytest.fixture(scope="function", autouse=False)
+def skip_pfcwd_higher_speeds(
+        request,
+        duthosts,
+        enum_rand_one_per_hwsku_frontend_hostname,
+        setup_pfc_test):
+    duthost = duthosts[enum_rand_one_per_hwsku_frontend_hostname]
+    if duthost.get_facts().get("asic_type") != "cisco-8000" or not duthost.get_facts().get("modular_chassis"):
```

Comment:
variable has_flakyness_400gbps_pfc_gen = ...

if true:
...

No yield and return